lib/localeapp/exception_handler.rb

```diff
@@ -1,16 +1,17 @@
 module Localeapp
   class ExceptionHandler
-    def self.call(exception, locale, key, options)
+    def self.call(exception, locale, key_or_keys, options)
+      keys = Array(key_or_keys).map { |key| ERB::Util.html_escape(key.to_s) }
       Localeapp.log(exception.message)
       # Which exact exception is set up by our i18n shims
       if exception.is_a? Localeapp::I18nMissingTranslationException
-        Localeapp.log("Detected missing translation for key(s) #{key.inspect}")
+        Localeapp.log("Detected missing translation for key(s) #{keys.inspect}")
 
-        [*key].each do |key|
+        keys.each do |key|
           Localeapp.missing_translations.add(locale, key, nil, options || {})
         end
 
-        [locale, key].join(', ')
+        [locale, keys].join('.')
```

Comment:
We should join with a comma, not a dot, because:
```ruby
# This is what we want
[:en, "foo.bar"].join(", ")           # => "en, foo.bar"
[:en, %w{foo.bar foo.baz}].join(", ") # => "en, foo.bar, foo.baz"

# This is what we don't want
[:en, "foo.bar"].join(".")            # => "en.foo.bar"
[:en, %w{foo.bar foo.baz}].join(".")  # => "en.foo.bar.foo.baz"
```

I should add a test for this :confused: